test/util/util.c

```diff
@@ -569,6 +569,13 @@ connect_with_tls (const char *endpoint,
       mongoc_stream_destroy (stream);
       stream = NULL;
    }
+
+   if (!mongoc_stream_tls_handshake_block (
```

Comment:
Fix for the csfle CLI tool. Omitting this explicit call to handshake the TLS stream was not an issue for OpenSSL, since it seemed to do the handshake on the first write. But it was a problem for secure transport.